src/common/kernel.cpp

```diff
@@ -143,6 +148,208 @@ static void dump_backtrace()
+        case 0xC000008F: errorString = "STATUS_FLOAT_INEXACT_RESULT"; break;
+        case 0xC0000090: errorString = "STATUS_FLOAT_INVALID_OPERATION"; break;
+        case 0xC0000091: errorString = "STATUS_FLOAT_OVERFLOW"; break;
+        case 0xC0000092: errorString = "STATUS_FLOAT_STACK_CHECK"; break;
+        case 0xC0000093: errorString = "STATUS_FLOAT_UNDERFLOW"; break;
+        case 0xC0000094: errorString = "STATUS_INTEGER_DIVIDE_BY_ZERO"; break;
+        case 0xC0000095: errorString = "STATUS_INTEGER_OVERFLOW"; break;
+        case 0xC0000096: errorString = "STATUS_PRIVILEGED_INSTRUCTION"; break;
+        case 0xC00000FD: errorString = "STATUS_STACK_OVERFLOW"; break;
+        case 0xC0000135: errorString = "STATUS_DLL_NOT_FOUND"; break;
+        case 0xC0000138: errorString = "STATUS_ORDINAL_NOT_FOUND"; break;
+        case 0xC0000139: errorString = "STATUS_ENTRYPOINT_NOT_FOUND"; break;
+        case 0xC000013A: errorString = "STATUS_CONTROL_C_EXIT"; break;
+        case 0xC0000142: errorString = "STATUS_DLL_INIT_FAILED"; break;
+        case 0xC00001B2: errorString = "STATUS_CONTROL_STACK_VIOLATION"; break;
+        case 0xC00002B4: errorString = "STATUS_FLOAT_MULTIPLE_FAULTS"; break;
+        case 0xC00002B5: errorString = "STATUS_FLOAT_MULTIPLE_TRAPS"; break;
+        case 0xC00002C9: errorString = "STATUS_REG_NAT_CONSUMPTION"; break;
+        case 0xC0000374: errorString = "STATUS_HEAP_CORRUPTION"; break;
+        case 0xC0000409: errorString = "STATUS_STACK_BUFFER_OVERRUN"; break;
+        case 0xC0000417: errorString = "STATUS_INVALID_CRUNTIME_PARAMETER"; break;
+        case 0xC0000420: errorString = "STATUS_ASSERTION_FAILURE"; break;
+        case 0xC00004A2: errorString = "STATUS_ENCLAVE_VIOLATION"; break;
+        case 0xC0000515: errorString = "STATUS_INTERRUPTED"; break;
+        case 0xC0000516: errorString = "STATUS_THREAD_NOT_RUNNING"; break;
+        case 0xC0000718: errorString = "STATUS_ALREADY_REGISTERED"; break;
+        case 0xC015000F: errorString = "STATUS_SXS_EARLY_DEACTIVATION"; break;
+        case 0xC0150010: errorString = "STATUS_SXS_INVALID_DEACTIVATION"; break;
+
+        // Other
+        case 0xE06D7363: errorString = "UNKNOWN_APPLICATION_ERROR"; break; // TODO: What is the name for 0XE06D7363?
+    }
+    // clang-format on
+
+    return fmt::format("{} ({:#08X})", errorString, errorCode);
+}
+
+// https://stackoverflow.com/a/50208684
+// From MSDN.WhiteKnight
+LONG WINAPI TopLevelExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo)
+{
+    // https://www.freelists.org/post/luajit/FirstChance-Exception-in-luajit,1
+    // https://love2d.org/forums/viewtopic.php?t=84336
+    switch (pExceptionInfo->ExceptionRecord->ExceptionCode)
+    {
+        // exceptions NOT documented by Microsoft
+        case 0xE24C4A02 : // Magic exception number from LuaJIT: Ignore!
+            return EXCEPTION_CONTINUE_SEARCH;
+
+        default:
+            break;
+    }
```

Comment:
Found the issue, and it's pretty interesting!

LuaJIT throws exceptions and swallows them as part of its regular operation. So if we hook onto them... yeah.

This block tells the handler to skip over any exceptions from LuaJIT. (We rely on exceptions and all bad things in Lua being swallowed anyway, so nothing is being lost here)

This also explains what all those random silent exceptions at startup and runtime are!

Reply:
Found another one that was encountered when saving a lua script with some bad copy-pasta, `0xE24C4A03`.  Adding it to the switch resolved.